src/cider/exec/nextgen/jitlib/llvmjit/LLVMJITFunction.cpp

```diff
@@ -222,7 +222,7 @@ JITValuePointer LLVMJITFunction::emitRuntimeFunctionCall(
 
 void LLVMJITFunction::cloneFunctionRecursive(llvm::Function* fn) {
   CHECK(fn);
-  if (!fn->isDeclaration()) {
+  if (!fn->isDeclaration() || !fn->hasFnAttribute(llvm::Attribute::AlwaysInline)) {
```

Comment:
Is this necessary change?  what will happen if w/o it?

Reply:
some external functions that cannot be inlined may core dumped here, so  just clone inline function and use NEVER_INLINE key words in such function declaration to skip it.